src/MSBuild/MSBuild.csproj

```diff
@@ -176,6 +176,8 @@
     <PackageReference Include="Microsoft.BuildXL.Processes" Condition="'$(FeatureReportFileAccesses)' == 'true'" PrivateAssets="all" />
     <PackageReference Include="Microsoft.IO.Redist" Condition="'$(FeatureMSIORedist)' == 'true'" />
     <PackageReference Include="System.Configuration.ConfigurationManager" />
+    <!-- Explicitly reference STJ in .NET 8 to bump to a patched version for the deps file -->
```

Comment:
Should this be flown to vs17.12? 17.12 is .NET 9

Reply:
Given we are green for CG in 17.12 - I wouldn't carry over this adjustment.
Let's skip this one

Reply:
No CG alert in 17.12. We don't need this.